### PR TITLE
New `--include` option for `jf rt search`

### PR DESCRIPTION
**Pull request by sarao1310**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: artifactory/commands/generic/generic.go [hunk 1]
```diff
@@ -93,3 +94,12 @@ func (gc *GenericCommand) SetDetailedSummary(detailedSummary bool) *GenericComma
 	gc.detailedSummary = detailedSummary
 	return gc
 }
+
+func (gc *GenericCommand) GetInclude() []string {
```
Comment (eyalbe4) [inline artifactory/commands/generic/generic.go:98]: Notice that non of the getter functions in this struct include the "Get" prefix in their names. Let's remove it here.
I actually suggest naming this function to `AqlInclue`.

File: artifactory/commands/generic/generic.go [hunk 2]
```diff
@@ -93,3 +94,12 @@ func (gc *GenericCommand) SetDetailedSummary(detailedSummary bool) *GenericComma
...
+	return gc.includedQueryPart
```
Comment (eyalbe4) [inline artifactory/commands/generic/generic.go:99]: includedQueryPart --> aqlInclude

File: artifactory/commands/generic/generic.go [hunk 3]
```diff
@@ -93,3 +94,12 @@ func (gc *GenericCommand) SetDetailedSummary(detailedSummary bool) *GenericComma
...
+}
+
+func (gc *GenericCommand) SetInclude(include []string) *GenericCommand {
```
Comment (eyalbe4) [inline artifactory/commands/generic/generic.go:102]: SetInclude --> SetAqlInclude

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Let's ensure that new `include` field support all the entities that AQL's `items.find` supports. The list is documented under the **Entities and Fields** section [here](https://jfrog.com/help/r/jfrog-artifactory-documentation/artifactory-query-language).

File: artifactory/commands/generic/generic.go (hunk 3 above)
Comment (sarao1310) [reply artifactory/commands/generic/generic.go:102]: Done

File: artifactory/commands/generic/generic.go (hunk 2 above)
Comment (sarao1310) [reply artifactory/commands/generic/generic.go:99]: Done

File: artifactory/commands/generic/generic.go (hunk 1 above)
Comment (sarao1310) [reply artifactory/commands/generic/generic.go:98]: Done

**Pull request merged**